programs/bpf/Cargo.lock

```diff
@@ -4664,6 +4664,7 @@ dependencies = [
  "chrono",
  "chrono-humanize",
  "crossbeam-channel",
+ "dashmap",
```

Comment:
why does bpf need dashmap?

Reply:
I was a bit confused by this as well. It seems that all the different solana packages are duplicated in this Cargo.lock, and that it's checked by the CI.

This is adding "dashmap" to "solana-ledger", which requires dashmap in `Blockstore::get_accounts_used_in_range"

Reply:
I misread this. dashmap was probably already used by something. Probably some function now has it in a pub signature and that causes it to get pulled in. As long as you're not adding a new cargo.toml entry for dashmap, I think this is correct.